include/LeMonADE/feature/FeatureExcludedVolumeBcc.h

```diff
@@ -0,0 +1,394 @@
+  o/.|.\o    E   nvironment    | LeMonADE Principal Developers (see AUTHORS)
+    ooo                        | 
+----------------------------------------------------------------------------------
+
+This file is part of LeMonADE.
+
+LeMonADE is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+LeMonADE is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with LeMonADE.  If not, see <http://www.gnu.org/licenses/>.
+
+--------------------------------------------------------------------------------*/
+
+#ifndef LEMONADE_FEATURE_FEATUREEXCLUDEDVOLUMEBCC_H
+#define LEMONADE_FEATURE_FEATUREEXCLUDEDVOLUMEBCC_H
+
+#include <LeMonADE/feature/Feature.h>
+#include <LeMonADE/feature/FeatureLattice.h>
+#include <LeMonADE/feature/FeatureLatticePowerOfTwo.h>
+
+#include <LeMonADE/updater/moves/MoveBase.h>
+#include <LeMonADE/updater/moves/MoveLocalBcc.h>
+#include <LeMonADE/updater/moves/MoveAddBccMonomer.h>
+
+
+/*****************************************************************************/
+/**
+ * @file
+ * @date   2016/02/18
+ * @author Ron and Martin
+ *
+ * @class FeatureExcludedVolumeBcc
+ * @brief This Feature adds excluded volume check to the system.
+ * It is specialised to the body centered cubic lattice.
+ * 
+ * @details The excluded volume is checked by a lattice occupation algorithm
+ * on a body centered cubic lattice.
+ * The feature is implemented as a class template, where the template parameters 
+ * \a SpecializedClass and \a ValueTypes specify, what
+ * type of lattice is used and which kind of value is saved on the lattice.
+ * This feature requires FeatureLattice.
+ * Notice: only implementations of Bcc moves included to avoid a mix of 
+ * FeatureExcludedVolumeBcc and MoveLocalSc.
+ *
+ * @tparam <SpecializedClass<ValueType>> name of the specialized class.
+ * Default is FeatureLattice.
+ * 
+ * @tparam <ValueType> type of the lattice value. Default is bool.
+ * */
+/*****************************************************************************/
+
+/**
+ * @brief Forward declaration. Equivalent to FeatureExcludedVolumeBcc< FeatureLattice <bool> >
+ */
+template<class SpecializedClass = FeatureLattice<> > class FeatureExcludedVolumeBcc;
+
+///////////////////////////////////////////////////////////////////////////////
+//DEFINITION OF THE CLASS TEMPLATE   	                                ///////
+//Implementation of the members below					///////
+///////////////////////////////////////////////////////////////////////////////
+
+template<template<typename> class SpecializedClass, typename ValueType>
```

Comment:
I change the typename _SpecializedClass_ to **LatticeClassType** and _ValueType_ to **LatticeValueType**. This seems to be self-explanatory.